ts/Extensions/ParallelCoordinates.ts

```diff
@@ -691,8 +691,31 @@ namespace ParallelAxis {
                 }
             });
 
-            axis.dataMin = arrayMin(currentPoints);
-            axis.dataMax = arrayMax(currentPoints);
+            // Take into account range series points as well (#15752)
+            let min: number = NaN,
+                max: number = NaN;
+
+            currentPoints.forEach(function (point): void {
+                if (isArray(point)) {
+                    max = arrayMax(point);
+                    min = arrayMin(point);
+                }
+            });
+
+            if (
+                !isArray(arrayMin(currentPoints)) ||
+                !isArray(arrayMax(currentPoints))
```

Comment:
`arrayMin` finds a minimum value in **array of values**. Here you're trying to find a minimum value of **array of arrays**

Reply:
Exactly, to check if any of the points is not an array (to take into account also points from series like line, column, etc.)

Reply:
But why using `arrayMin` for that? Can't you just do `currentPoints[0]`?

I mean, comparing arrays using `arrayMin` is semantically wrong. If you want to distinguish if the point is a value (line, column etc.) or array (range series), then you should do it differently in my opinion

Reply:
I took a closer look and mixed up non-range and range series: http://jsfiddle.net/BlackLabel/kLhc5j0z/

We can't use `currentPoints[0]`, it was just a blind shot, forget about it.
But I still think that we shouldn't use `arrayMin` in this specific use case. I see you're still working on it so please let me know when it's ready to review ;)

Reply:
Ok, you're right. This solution to the problem with mixed non-range and range series is not the best. In fact, I've got another solution.